massgov/mayflower
# DP-22263 Implement responsive images for promotional page images in key message sections

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
Adjusted the key message component so it uses responsive images instead.

## Related Issue / Ticket

- https://jira.mass.gov/browse/DP-22263
- https://github.com/massgov/openmass/pull/864

## Steps to Test


1. Go to qagcampaign-landing-with-image-key-message-header
2. make sure picture element is in use and that the key message and key message section images are visible.

## Screenshots
<img width="1481" alt="Captura de Pantalla 2021-07-21 a la(s) 14 41 32" src="https://user-images.githubusercontent.com/141685/126536535-2bb983e8-483d-4c01-96cc-998d968b3d6a.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Key Message
* Key Message Section

#### @TODO


*

#### Today I learned...
